src/io/FileTransaction.hpp

```diff
@@ -65,6 +65,13 @@ class FileTransaction {
   }
 
   /**
+   * Returns the final path.
+   */
+  Path GetFinalPath() const {
```

Comment:
Why? This was given to the constructor. So the caller already knows the final path.

Reply:
removed...
The caller has build this name in an additional function, so it was an attempt to extend the component...
But of course I can split the calling line.